### ISSUE TITLE
CUDA Instructions to be included in README.md

### ISSUE THREAD

**Issue by pligor** [h2oai/h2o-llmstudio#243]
### 📃 Documentation

Install Conda

conda create --name <environment_name> python=3.10

Install CUDA

#conda install cuda -c nvidia/label/cuda-11.8.0
conda install cuda -c nvidia/label/cuda-11.7.1

Install CuDNN

#usually already installed

python3 -m pip install nvidia-cudnn-cu11

Install 

#conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia

conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia

Test Cuda Installation

python -c "import torch; print(torch.cuda.is_available())";

### Motivation

All the "before" steps were missing. Providing here the ones that seem most prominent. If you include a section like so in the README.md file would unblock lots of users and developers
Many thanks

**Comment by pascal-pfeiffer** [comment]
Please follow the recommended install steps (https://github.com/h2oai/h2o-llmstudio#recommended-install). There is also a part about installing NVIDIA Drivers and CUDA on system level (https://github.com/h2oai/h2o-llmstudio#installing-nvidia-drivers-if-required)

The CUDA installation steps you show above may or may not work properly when using a virtual environment or a docker. Pytorch itself comes with CUDA, only bitsandbytes will check for your system CUDA and this will hopefully change soon, too.
